server/build.gradle.kts

```diff
@@ -36,9 +36,14 @@ import io.spine.internal.gradle.publishing.gitHubPackages
 
 plugins {
     // Add the Gradle plugin for bootstrapping projects built with Spine.
-    // See: https://github.com/SpineEventEngine/bootstrap
+    // See: https://github.com/SpineEventEngine/bootstrap.
     id("io.spine.tools.gradle.bootstrap").version("1.9.0")
+
+    // The ShadowJar plugin ID has been updated to `com.gradleup.shadow`,
+    // but only versions from 8.3.0 onward are available.
+    // Since version 6.1.0 is required, the old plugin ID is used.
```

Comment:
As per our discussion, let's have a single place in which we could document the whole thing about this Shadow JAR plugin. And then reference this "place" when in need.